### PR TITLE
do not use protocols.template as a default template. Only when runnin…

### PR DESCRIPTION
**Pull request by pconesa**
…g config mode.

### CODE REVIEW

**Comment by dmaluenda** [comment]
Just a question to understand the intention. After this PR, the protocols' tree will be populated according to the plugins, including also the sections. Does it mean that if none of the plugins of my Scipion have (let's say) the CTF section? then my Scipion will not show that section until I install some plugin with that section defined therein. If so, it would be great!

Just a comment, the protocols.template is edited in this PR when it should be removed, isn't it?

File: scipion/scripts/config.py [hunk 1]
```diff
@@ -123,6 +123,10 @@ def main(args=None):
 
         scipionConfigFile = os.environ[SCIPION_CONFIG]
         # Global installation configuration files.
+
+        # NOTE: generating a protocols.conf form the template does not make much sense
```
Comment (delarosatrevin) [inline scipion/scripts/config.py:127]: type: .. from the template

**Comment by pconesa** [comment]
@dmaluenda , you are right. I'll try to summarize.

if there is a protocols.conf in the config folder it will take priority and will be loaded first. So it will decide sections and the order of those, etc

Then priority plugins are loaded, this are coming from an environment variable: now scipion-app defines it with pwem, tomo and pwchem (core plugins centralizing the core structure of the protocol.conf for their respective "domain".

Then rest of the plugins are loaded.

So, if there isn't any plugin, and not protocols.conf the tree will be empty.

**Comment by pconesa** [comment]
@dmaluenda , your wish is not granted. pwem is defining empty sections and they will be show, but I guess pyworkflow could do a cleanup at then end and not showing empty sections.

**Comment by dmaluenda** [comment]
Well, pwem is a plugin, thus it is (strictly speaking) right

**Comment by dmaluenda** [comment]
Even though, it's not a bad idea to remove empty sections

**Review by azazellochg — APPROVED** [review]

**Pull request merged**